tests/src/Traits/EntityTrait.php

```diff
@@ -49,7 +55,37 @@ protected function getEntityByLabel($entity_type, $label, $bundle = NULL) {
       return $storage->load($result);
     }
 
-    throw new \Exception("The entity with label '$label' was not found.");
+    throw new \RuntimeException("The entity with label '$label' was not found.");
+  }
+
+  /**
+   * Mapping of human readable entity type names to machine names.
+   *
+   * @return array
+   *   The entity type mapping.
+   */
+  protected static function entityTypeAliases() {
+    return [
+      'content' => 'node',
+    ];
+  }
+
+  /**
+   * Translates human readable entity types to machine names.
+   *
+   * @param string $entity_type
```

Comment:
Let's rename to `$entity_type_label` to make it clear.

Reply:
Renamed to `$entity_type_id` and also updated the trait to use strict typing.